pkg/controlplane/control/peer.go

```diff
@@ -138,6 +139,8 @@ func (m *peerMonitor) Start() {
 		if strikeCount < threshold {
 			<-ticker.C
 			continue
+		} else if heartbeatErr != nil {
+			m.logger.Errorf("Fail to sent heartbeat to %s : %v", m.pr.Name, heartbeatErr)
```

Comment:
```suggestion
			m.logger.Errorf("Failed to send heartbeat to %s: %v", m.pr.Name, heartbeatErr)
```
Would be better English